components/markdown_image_expand/markdown_image_expand.scss

```diff
@@ -0,0 +1,77 @@
+@import 'compass/css3';
+@import 'sass/utils/functions';
+@import 'utils/variables';
+
+.markdown-image-expand {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+}
+
+.markdown-image-expand__collapse-button {
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  display: none;
+  width: 24px;
+  height: 24px;
+  border: 1px solid var(--center-channel-color-16);
+  padding: 0;
+
+  font-size: 18px;
+  line-height: 21px;
+  color: var(--center-channel-color-56);
+
+  background-color: var(--center-channel-bg);
+  border-radius: 16px;
+  @include box-shadow($elevation-1);
+
+  .icon {
+    width: 18px;
+    height: 18px;
+  }
+
+  .markdown-image-expand--expanded:hover & {
+    display: block;
+  }
+
+  &:hover {
+    border: 1px solid var(--center-channel-color-24);
+
+    @include box-shadow($elevation-2);
```

Comment:
For some reason `$elevation-2` is rendering the shadow opacity at `0.012` and it should be rendering as `0.12`

Reply:
Alright. I'll fix this. By the way, all of the elevations starting from 2 to 6 have an opacity like this. Probably we have here some sort of wide spread issue.

Reply:
Seems like a mistake to me. Not sure when this was added

Reply:
Fixed only `$elevation-2` for now. 

@asaadmahmood I am sorry to bother you but according to Git blame you were the author of these `elevation` settings. May I request some assistance from you - are these opacity values incorrect or there were some reasoning behind this?

Reply:
@TheDarkestDay The opacity values are updated on master.
I did them on this PR:
https://github.com/mattermost/mattermost-webapp/pull/7696/files